portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DefinitionsSearch.testcase

```diff
@@ -79,4 +88,83 @@ definition {
 		ACDefinitions.noViewAddSearchQueryButton(query = "Test Query 5");
 	}
 
+	@description = "Feature ID: LRAC-13934 | Automation ID: LRAC-14680 | Test Summary: Customized query string will still collect search term"
+	@priority = 4
+	test ViewKeywordSearchedNewStringParameter {
+		task ("Create a new query string parameter") {
+			ACDefinitions.addSearchQuerySuccessfully(query = "query");
+
+			ACDefinitions.goToDefinitionType(definitionType = "Search");
+
+			ACDefinitions.viewSearchQuery(query = "query");
+		}
+
+		task ("Connect DXP to Analytics Cloud") {
+			ACSettings.takeTokenFromAnalyticsCloud();
+
+			TestCase.setUpPortalInstance();
+
+			ACUtils.launchDXP();
+
+			ACUtils.addSiteAndPage();
+
+			var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+		}
+
+		task ("Update keyword parameter on DXP") {
+			Navigator.openURL();
+
+			JSONLayout.addPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Search Page");
```

Comment:
Hey @joshitagaki, I think you don't need to create a new page here. You can use the AC Page that you created [here](https://github.com/liferay-ac/liferay-portal/pull/735/files#diff-83fef259c7778902d9e334672bbfb1ebfd03d8c121bd5fb87aa09c64193de939R109)

Reply:
Thanks! Forgot I made a page in the previous step